src/appleseed/renderer/modeling/bsdf/kelemenbrdf.cpp

```diff
@@ -607,8 +648,10 @@ namespace
             const double        m,
             const Spectrum&     rs)
         {
-            generate_specular_albedo_plot_data("Ward", m, WardMDF<double>(m), rs);
-            generate_specular_albedo_plot_data("Beckmann", m, BeckmannMDF<double>(m), rs);
+            generate_specular_albedo_plot_data("Ward", m, WardMDFAdapter<double>(m), rs);
+
+            // TODO: not really sure what to do with this... (est.)
```

Comment:
Beckmann cannot be sampled anymore using the old methods.
I'd need to recover the old implementation from the repository and copy the code here.
It is worth it?